.rubocop.yml

```diff
@@ -1,12 +1,17 @@
-Metrics/LineLength:
+AllCops:
+  TargetRubyVersion: 2.5
+  NewCops: disable
```

Comment:
Will we be re-enabling these later? They seem useful.

Reply:
I wasn't enthusiastic about either specifying `NewCops: enable` and fixing things every Rubocop upgrade, or individually enabling all the current cops.  But maybe the former isn't so bad?

Reply:
My reading of the new option is that if you specify neither, it won't enable the new cops but will warn you that there are new cops to explicitly enable/disable?

Reply:
I'd support either one though, so we eventually get improvements from new cops.

Reply:
The long list of warnings from specifying neither was pretty annoying, so I went with specifying `enable`.